packages/shared/src/medical/fhir/observations.ts

```diff
@@ -0,0 +1,58 @@
+import { Observation } from "@medplum/fhirtypes";
+
+// Not sure the names of the codes are correct,
+// but they're related and should be grouped together regardless
+const oxygenSaturationLoinc = {
+  SpO2: "2708-6",
+  Sp02ByPulseOximetry: "59408-5",
+  Sp02ByDirectSampling: "20509-6",
+  Sp02InCapillaryBloodByPulseOximetry: "59466-1",
+  Sp02OnRoomAir: "59410-1",
+} as const;
+
+const vitalsLoinc = {
+  HeartRate: "8867-4",
+  BloodPressureSystolic: "8480-6",
+  BloodPressureDiastolic: "8462-4",
+  RespiratoryRate: "9279-1",
+  InhaledOxygenFlowRate: "3151-8",
+  Temperature: "8310-5",
+  ...oxygenSaturationLoinc,
+  Weight: "29463-7",
+  Height: "8302-2",
+  BMI: "39156-5",
+} as const;
+
+// Create a type from the values
```

Comment:
nit: no need for this comment; we only add comments to the code if it's to explain WHY something it the way it is - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#493564ff82fa45edb294b6a729e94fba).